### ISSUE TITLE
Get-AzPolicyAssignment - Missing information about removed output

### ISSUE THREAD

**Issue by jagiraud** [MicrosoftDocs/azure-docs-powershell#3029]
### Type of issue

Missing information

### Feedback

Undocumented breaking change for the Get-AzPolicyAssignment cmdlet
https://learn.microsoft.com/en-us/powershell/azure/migrate-az-12.0.0?view=azps-12.0.0#get-azpolicyassignment

`Get-AzPolicyAssignment` no longer contains the `Identity` output. Causing the variable $objectId to be empty. 

## Previous behaviour
```powershell
$assignment = Get-AzPolicyAssignment
$objectId = $assignment.Identity.PrincipalId
```

## Current behaviour

```powershell
$assignment = Get-AzPolicyAssignment
$objectId = $assignment.IdentityPrincipalId
```

### Page URL

https://learn.microsoft.com/en-us/powershell/azure/migrate-az-12.0.0?view=azps-12.0.0#azresources

### Content source URL

https://github.com/MicrosoftDocs/azure-docs-powershell/blob/main/docs-conceptual/azps-12.0.0/migrate-az-12.0.0.md

### Author

@mikefrobbins

### Document Id

ba052cde-c7e2-77cf-3447-84a8a2e290ce

**Comment by TPavanBalaji** [comment]
@jagiraud 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by AjayBathini-MSFT** [comment]
@jagiraud 
Thanks for bringing this to our attention.
I'm going to assign this to the document author so they can take a look at it accordingly.

**Comment by mikefrobbins** [comment]
@jagiraud Thank you for reporting this oversight. I've contacted the Azure PowerShell engineering team for clarification. This issue is resolved via PR #3031.
